From 2bbe5cdcfedc3aa571e87fbd65f866350a83d815 Mon Sep 17 00:00:00 2001
From: "Le.Ma" <Le.Ma@amd.com>
Date: Fri, 8 Dec 2017 11:00:18 +0800
Subject: [PATCH 2834/4131] drm/amdkcl: [4.14] Fix to use
 drm_atomic_helper_wait_for_vblanks()

This fixes patch "drm/amd/display: Switch to drm_atomic_helper_wait_for_flip_done"

Change-Id: Ib1bf923195df542f28a3cd68e48e1f9eb92f3eca
Signed-off-by: Le.Ma <Le.Ma@amd.com>
Reviewed-by: Xiaojie Yuan <Xiaojie.Yuan@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
index 3a514a2..11e90d9 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm.c
@@ -4764,7 +4764,11 @@ static void amdgpu_dm_atomic_commit_tail(struct drm_atomic_state *state)
 #endif
 
 	if (wait_for_vblank)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 0)
+		drm_atomic_helper_wait_for_vblanks(dev, state);
+#else
 		drm_atomic_helper_wait_for_flip_done(dev, state);
+#endif
 
 	drm_atomic_helper_cleanup_planes(dev, state);
 }
-- 
2.7.4

